features/step_definitions/sample_file_steps.rb

```diff
@@ -0,0 +1,219 @@
+Given(/^a demo directory with a smelly file$/) do
+  contents = <<-EOS
+class Dirty
```

Comment:
I'd like to, but we run into trouble with the yaml files further down. What's your favorite heredoc unindenter?